rhwp/rhwp-studio/src/ui/agent-sidebar/cloud-ui.ts

```
@@ -0,0 +1,1069 @@
+    }
+    if (localTurnPending) {
+      panelStatus.textContent = '현재 응답이 끝나면 클라우드로 옮깁니다.';
+      panelDetail.textContent = '앱을 닫으면 전송 확인이 끝날 때까지 기다립니다.';
+      panelActions.append(action('전송 예약 취소', deps.onCancelPendingTransfer));
+      return;
+    }
+    switch (session.kind) {
+      case 'idle':
+        const profileReady = snapshot.profile.kind === 'configured' && snapshot.profile.connection === 'ready';
+        const appHosted = snapshot.profile.kind === 'configured' && snapshot.profile.mode === 'app-hosted';
+        const appHostedLock = managedCloudLock(snapshot);
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)